dists/go-ipfs/mkdist.js

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env node
```

Comment:
im not talking about this shebang, i'm talking about not forcing gulp for everything. I'm taling about bringing back _this file_.

---

Fwiw, 
- the shebang is standard and tons of modules use it, see: https://github.com/search?q=%22%23%21%2Fusr%2Fbin%2Fenv+node%22&ref=searchresults&type=Code&utf8=%E2%9C%93
- but the shebangs are _known to break on windows_, so it's not properly cross platform.
- but having node as `nodejs` is orthogonal to this. 
- and a bin called `nodejs` is going to break with the way that the entire node ecosystem calls node, from packages and from npm `package.json` scripts (by calling `node $args` directly). 
- beyond this, should probably file a bug with arch linux. i'm sure they have reasons to call it `nodejs` but it does break tons of shit. :(